### ISSUE TITLE
Feature - Save the configuration with models in alphabetical order

### ISSUE THREAD

**Issue by b-per** [ciejer/tangata_local#42]
It would be good for tangata to provide refactoring options. 

It could change the existing `yml` files to sort them by model name and save new models in the correct alphabetical order. Having models in that order make it much easier to edit the `yml` files manually.

**Comment by ciejer** [comment]
Ooh I like this one - and might be doable (with yaml comments remaining in place even perhaps!).

I'm thinking on by default with ability to turn off in config?

**Comment by b-per** [comment]
On by default looks good to me! I played a bit with `ruamel.yaml` yesterday and this is what I got to order the models:
```
yml['models'] = sorted(yml['models'], key=lambda model: model['name'])
```

**Comment by ciejer** [comment]
Released in 0.2.0
